tidy3d/components/data/monitor_data.py

```diff
@@ -1363,6 +1363,7 @@ def modes_info(self) -> xr.Dataset:
             "wg TM fraction": self.pol_fraction_waveguide["tm"],
             "mode area": self.mode_area,
             "group index": self.n_group_raw,  # Use raw field to avoid issuing a warning
+            "dispersion (ps/(nm km))": self.dispersion_raw,  # Use raw field to avoid issuing a warning
```

Comment:
if we're including units here, maybe put in the other ones? or does this break backwards compatibility?

Reply:
Rigorously, it does break backwards compatibility, but I wouldn't think this is critical in any way. It is still quite new and probably used only in post-processing.

My guess for the original lack of units is that we only added units to fields that don't follow Tidy3D's standard units (µm, s), so mode area is implied to be in μm² and wavelength in μm. Although I'd prefer to have units everywhere, I don't really like the breaking change.

What do you think?

Reply:
that's fine with me.